src/date-picker/__integ__/date-picker-embedded.test.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+import DatePickerPage from './page-objects/date-picker-page';
+import createWrapper from '../../../lib/components/test-utils/selectors';
+
+describe('Date picker embedded interactions', () => {
+  const setupTest = (testFn: (page: DatePickerPage) => Promise<void>) => {
+    return useBrowser(async browser => {
+      const page = new DatePickerPage(createWrapper().findDatePicker().getElement(), browser);
+      await browser.url('#/light/date-picker/embedded');
+      await testFn(page);
+    });
+  };
+
+  test(
```

Comment:
I would make the tests a bit more general. The first one can start from focusable-element-before and tab until the focusable-element-after. The second one -- vice versa.

Also, there is certain focusing logic still inside the calendar which is focusing the date buttons when using keyboard navigation. It would be nice to replicate the following test here: https://github.com/cloudscape-design/components/blob/main/src/date-picker/__integ__/date-picker.test.ts#L107